File: src/client/classes/api/req_res/get-notifications-by-target-id-response.ts
'use strict'

import { Notification } from '@/classes/datas/notification'
import { GkillAPIResponse } from '../gkill-api-response'

export class GetNotificationsByTargetIDResponse extends GkillAPIResponse {

    notifications: Array<Notification>

    constructor() {
        super()
        this.notifications = new Array<Notification>()
    }

}

File: src/server/gkill/api/req_res/get_notifications_by_target_id_response.go
package req_res

import (
	"github.com/mt3hr/gkill/src/server/gkill/api/message"
	"github.com/mt3hr/gkill/src/server/gkill/dao/reps"
)

type GetNotificationsByTargetIDResponse struct {
	Messages message.GkillMessages `json:"messages"`

	Errors message.GkillErrors `json:"errors"`

	Notifications []reps.Notification `json:"notifications"`
}
